packages/design-system/src/components/datalist/searchInput.js

```diff
@@ -88,6 +88,17 @@ const Input = styled.input.attrs({
     display: none;
   }
 
+  &:placeholder-shown {
+    text-overflow: ellipsis;
+  }
+  &::-webkit-input-placeholder {
+    /* Chrome/Opera/Safari */
+    text-overflow: ellipsis;
+  }
+  &::-moz-placeholder {
+    /* Firefox 19+ */
+    text-overflow: ellipsis;
+  }
```

Comment:
This does not seem to be working, text is cut off without ellipsis. Can you check this again?

These might also need `white-space: nowrap; overflow: hidden;`

Reply:
I can look into that.  It should be working in Chrome when the `input` doesn't have focus but when an `input` is in focus Chrome disregards the `text-overflow`.  Firefox seems to allow it and I haven't tested it on Safari.

Reply:
It looks like theres no way to do this for focused inputs in Chrome. But as I mentioned it works when the search is not in focus for Chrome, and it works on Firefox for both in focus and not.

https://user-images.githubusercontent.com/14057928/173126206-d5d1a210-6c8f-438a-9266-9a7d32ef87aa.mp4